lib/reports/establishments/index.js

```diff
@@ -43,10 +44,21 @@ module.exports = ({ db }) => {
             // onVal isn't currently documented but it was added here: https://github.com/knex/knex/pull/2746
             b2.onVal('roles.type', '=', 'pelh')
               .orOnVal('roles.type', '=', 'nprc');
-          });
+          })
+          .andOnNull('roles.deleted');
       })
       .leftJoin('profiles AS pelh', 'pelh.id', 'roles.profile_id')
 
+      // HOLCs
+      .leftJoin('roles AS roles2', builder => {
+        builder.on('roles2.establishment_id', '=', 'est.id')
+          .andOn(b2 => {
+            b2.onVal('roles2.type', '=', 'holc');
```

Comment:
Is there a reason for nesting this inside a function?

Reply:
Probably not, was a cut n paste from above